app/components/osc-webhook-triggers/osc-webhook-triggers.html

```diff
@@ -80,10 +80,10 @@
   </div>
 
   <div class="add-webhook-action-btns">
-    <button class="btn btn-link pad-left-none" type="button" ng-click="$ctrl.addEmptyWebhookTrigger()">Add Webhook</button>
+    <a href="" role="button" ng-click="$ctrl.addEmptyWebhookTrigger()">Add Webhook</a>
```

Comment:
We want to use `<button>` instead of `<a>` because it's better for accessibility. It's also prevents "open in a new window" when we know it won't work.

Reply:
ok, should all the `<a>` be converted to `<button>`s

Reply:
or should I make the change so that .action-divider can work with either?

Reply:
I wouldn’t change them to buttons yet. A little risky so close to the end of the release. Eventually we want to though